galley/pkg/config/analysis/context.go

```diff
@@ -39,4 +39,7 @@ type Context interface {
 
 	// Canceled indicates that the context has been canceled. The analyzer should stop executing as soon as possible.
 	Canceled() bool
+
+	// Disabled reports if a collection is disabled.
+	Disabled(c collection.Name) bool
```

Comment:
Technically, you don't need in the context. This is only needed for the aggregated/combined analyzers. It maybe better to special case that and avoid introducing this API for the analyzers in general.

Reply:
Alright, CombinedAnalyzer now handles this special case without needing to add anything to Context.